src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -363,77 +375,84 @@ FullyConnected_bf_tiled::GetAutoTuneParams(const fully_connected_params& params,
             // Tuning for Meteor Lake
             if (is_weight_with_small_ofm(params, output_f)) {
                 if (params.weights.GetLayout() == WeightsLayout::os_is_yx_osv32_isv2) {
-                    return selector.Default(tune_params(1, 1, 4, 2, 1, 1, EXE_MODE_DEFAULT));
+                    return selector.Default(tune_params(1, 1, 4, 2, 1, 1, 1, EXE_MODE_DEFAULT));
                 } else if (params.weights.GetLayout() == WeightsLayout::os_iyx_osv16) {
-                    return selector.Default(tune_params(1, 1, 4, 4, 1, 1, EXE_MODE_DEFAULT));
+                    return selector.Default(tune_params(1, 1, 4, 4, 1, 1, 1, EXE_MODE_DEFAULT));
                 }
             } else {
                 if (params.weights.GetLayout() == WeightsLayout::os_iyx_osv16) {
-                    return selector.Default(tune_params(1, 1, 4, 4, 1, 1, EXE_MODE_DEFAULT));
+                    return selector.Default(tune_params(1, 1, 4, 4, 1, 1, 1, EXE_MODE_DEFAULT));
+                } else if (params.weights.GetLayout() == WeightsLayout::os_is_yx_osv64_isv2) {
+                    if (is_suitable_outer_ofm(output_f))
```

Comment:
I'd like to suggest to add this check in the validation of tuning param.

Reply:
applied